spec/controllers/users_controller_spec.rb

```diff
@@ -36,6 +36,20 @@
     end
   end
 
+  describe 'PUT #update' do
```

Comment:
add test for case when user updates existing address. You will see that your functionality does not work